src/Model/Callback/JobStatus.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenEuropa\CdtClient\Model\Callback;
+
+/**
+ * Class JobStatus.
+ *
+ * Represents the single callback sent to the defined endpoint
+ * @todo all properties are Uppercase in the request.
+ */
+class JobStatus
```

Comment:
The DTOs that are not implemented don't have getters and setters yet, for the sake of readability. I'll generate them with the next PRs.